include/ginkgo/ginkgo.hpp

```diff
@@ -125,11 +125,9 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <ginkgo/core/solver/gmres.hpp>
 #include <ginkgo/core/solver/idr.hpp>
 #include <ginkgo/core/solver/ir.hpp>
-#include <ginkgo/core/solver/lower_trs.hpp>
 #include <ginkgo/core/solver/multigrid.hpp>
 #include <ginkgo/core/solver/solver_base.hpp>
 #include <ginkgo/core/solver/solver_traits.hpp>
-#include <ginkgo/core/solver/upper_trs.hpp>
```

Comment:
Shouldn't these be kept?

Reply:
These are moved to `triangular.hpp` in develop, after rebasing that one should be here.

Reply:
I don't think I will rebase this PR or #1054, so I will fix it